mailpoet/tests/DataGenerator/Generators/WooCommercePastRevenues.php

```diff
@@ -343,55 +367,57 @@ private function createProduct($name, $price, $categoryIds = [], $discount = 0)
   /**
    * @return array
    */
-  private function createSentEmailData(NewsletterEntity $newsletter, $sentAt, $subscribersIds, $segmentId) {
+  private function createSentEmailData(NewsletterEntity $newsletter, string $sentAt, array $subscribersIds, $segmentId) {
+    $connection = $this->entityManager->getConnection();
+
     // Sending task
-    $task = ScheduledTask::createOrUpdate([
-      'type' => Sending::TASK_TYPE,
-      'status' => ScheduledTask::STATUS_COMPLETED,
-      'created_at' => $sentAt,
-      'processed_at' => $sentAt,
-    ]);
-    $task->save();
+    $task = new ScheduledTaskEntity();
+    $task->setType(Sending::TASK_TYPE);
+    $task->setStatus(ScheduledTaskEntity::STATUS_COMPLETED);
+    $task->setCreatedAt(new Carbon($sentAt));
+    $task->setProcessedAt(new Carbon($sentAt));
+    $this->scheduledTasksRepository->persist($task);
+    $this->scheduledTasksRepository->flush();
 
     // Sending queue
-    $queue = SendingQueue::createOrUpdate([
-      'task_id' => $task->id,
-      'newsletter_id' => $newsletter->getId(),
-      'count_total' => count($subscribersIds),
-      'count_processed' => count($subscribersIds),
-    ]);
-    $queue->save();
+    $queue = new SendingQueueEntity();
```

Comment:
`MailPoet\Test\DataFactories\SendingQueue` can facilitate here

Reply:
Similar to the above, I don't think there is a way to use the SendingQueue data factory to set `countTotal` and `countProcessed`.